libs/design/src/atoms/form/radio/radio.md

```diff
@@ -0,0 +1,55 @@
+# DaffRadio and DaffRadioSet
+The `DaffRadioComponent` is used to select a single value from a selection of values. It can be hooked into Angular's `FormControl` to accomodate custom functionality. The `DaffRadioSetComponent` serves as a wrapper around a logical group of radios to provide styling.
+
+## Accessibility
+The `DaffRadioComponent` uses native `<input>` HTML elements to ensure an accesible experience by default. It supports inputs to customize the experience for accessibility by allowing native input for `aria-label` and `aria-labelledby`.
+
+## DaffRadio Inputs
+
+Property | Name            |  Type   |                                                                                         Description
+---------|------- | :-----: | ---------------------------------------------------------------------------:
+Input | checked | boolean | The value representing whether the radio is checked. default is equal to `false`|
+Input | value   | string  | The value of the radio | 
+Input |  id | string | The id of the radio. It is uniquely generated but can be overwritten by the user. Must be unique |
+Input |  name | string | The name of the radio |
+Input |  label | string | The label for the radio. Used for setting `aria-label` attribute. Defaults to equal name if nothing is provided. |
+Output | selectionChange | boolean | An event that emits everytime a selection has been changed.|
+
+
+
+## DaffRadioSet Properties
```

Comment:
`Radio Set Properties`

Reply:
These are now fixed in 428d88c9ff856cf8fedf81509a513daaebfb38e1.